dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -36,6 +36,12 @@ service AppCallback {
   // User application can save the states or send the events to the output
   // bindings optionally by returning BindingEventResponse.
   rpc OnBindingEvent(BindingEventRequest) returns (BindingEventResponse) {}
+
+  // Listens configuration update events from the configuration store.
+  rpc OnConfigurationEvent(ConfigurationEventRequest) returns (google.protobuf.Empty) {}
+
+  // Get effective configuration in app. Daprd will call this method to do inspection.
+  rpc GetEffectiveConfiguration(google.protobuf.Empty) returns (GetEffectiveConfigurationResponse) {}
```

Comment:
What is the need for this? Can you give an example? I am concerned about adding methods to the callback API if not for a strong need.